tools/output-number-of-buffer-segments.rb

```diff
@@ -0,0 +1,31 @@
+#!/bin/ruby
+
+require "open3"
+require "json"
+
+if ARGV.size() == 0
+  puts "Usage: $0 DATABASE_PATH"
+  puts " e.g.: $0 ~/database/db"
+  exit
+end
+
+DATABASE_PATH = ARGV[0]
+stdout, stderr, status = Open3.capture3("groonga",
+                                        DATABASE_PATH,
+                                        "dump",
+                                        "--dump_plugins","no",
+                                        "--dump_schema", "no",
+                                        "--dump_records", "no",
+                                        "--dump_configs", "no")
+stdout.each_line do |line|
+  index_name = "#{line.split()[1]}.#{line.split()[2]}"
+  stdout, stderr, status = Open3.capture3("groonga",
+                                          DATABASE_PATH,
+                                          "object_inspect",
+                                          index_name)
+  unix_time_when_command_executed = JSON.parse(stdout)[0][1]
```

Comment:
How about check the status and output stdout and stderr if an error occurred?

Reply:
Thank you for your comment.
I see. I've added them at e227389.